depends/packages/libmultiprocess.mk

```diff
@@ -4,6 +4,16 @@ $(package)_download_path=$(native_$(package)_download_path)
 $(package)_file_name=$(native_$(package)_file_name)
 $(package)_sha256_hash=$(native_$(package)_sha256_hash)
 $(package)_dependencies=native_$(package) capnp
+ifneq ($(host),$(build))
+$(package)_dependencies += native_capnp
+endif
+
+define $(package)_set_vars :=
+ifneq ($(host),$(build))
+$(package)_cmake_opts := -DCAPNP_EXECUTABLE="$$(native_capnp_prefixbin)/capnp"
+$(package)_cmake_opts += -DCAPNPC_CXX_EXECUTABLE="$$(native_capnp_prefixbin)/capnpc-c++"
```

Comment:
In commit "build: Fix `libmultiprocess` cross-compiling to Linux hosts" (c0f5cc14ef9fae2b2de4222ee061729629ebb6b4)

Note (just for understanding): This overrides the capnp tool paths used by the cross-compiled libmultiprocess build, so it will use native capnp tools, instead the cross compiled ones found by `find_package(CapnProto)` in the libmultiprocess build, which is needed to find the cross-compiled capnp libraries, but also finds the cross-compiled tools.

There's some documentation about overriding these variables: https://github.com/capnproto/capnproto/blob/55a368beb987abf9eeb9b3843e9c5423ad37ab29/c%2B%2B/cmake/CapnProtoConfig.cmake.in#L18-L22